src/cpp/r/include/r/session/RSession.hpp

```diff
@@ -173,19 +173,25 @@ void reportAndLogWarning(const std::string& warning);
 
 // suspend/resume
 bool isSuspendable(const std::string& prompt);
-bool suspend(bool force, int status = EXIT_SUCCESS);
+bool suspend(bool force, int status, const std::string& envVarSaveBlacklist);
 
 struct RSuspendOptions
 {
    RSuspendOptions(int exitStatus)
-      : status(exitStatus), saveMinimal(false), saveWorkspace(false), 
-        excludePackages(false)
+      : status(exitStatus)
+   {
+   }
+
+   RSuspendOptions(int exitStatus, const std::string& blacklist) 
+      : status(exitStatus),
+        envVarSaveBlacklist(blacklist)
    {
    }
    int status;
```

Comment:
It looks like `status` is no longer initialized to `EXIT_SUCCESS`?

Reply:
(although not a big deal since it looks like there's currently no constructor that doesn't set it!)